.github/workflows/e2e_integration_test.yml

```diff
@@ -25,6 +25,7 @@ jobs:
     env:
       DISTRIBUTION_PUBLIC_KEY: ${{ vars.DISTRIBUTION_PUBLIC_KEY }}
       DISTRIBUTION_SEED: ${{ vars.DISTRIBUTION_SEED }}
+      CHANNEL_ACCOUNT_ENCRYPTION_PASSPHRASE: ${{ vars.DISTRIBUTION_SEED }}
```

Comment:
Wasn't this being added on the last PR?

Reply:
This addition was missing in the e2e docker compose file.